Makefile

```diff
@@ -76,13 +76,16 @@ test: test/go test/web
 test/go: COVERAGE ?= false
 test/go: COVERAGE_OPTS ?= -covermode=atomic
 test/go: COVERAGE_OUTPUT ?= coverage.out
+test/go: setup-envtest
+test/go: ENVTEST_BIN ?= ${PWD}/.dev/bin
```

Comment:
```suggestion
test/go: ENVTEST_BIN ?= ./.dev/bin
```
This should be fine

Reply:
@khanhtc1202 
We need it as absolute path for setup-envtest. So I want to keep it as is.
I added the comment.

[4b1f396](https://github.com/pipe-cd/pipecd/pull/5399/commits/4b1f3962c6444f1353d0ab7119a80b4d8f888dad)

Reply:
Got it and reapproved 👍